samples/server/go-server/api/server/storage/storager_test.go

```diff
@@ -444,10 +444,13 @@ func doTestStorager(t *testing.T, createStore func(t *testing.T) (server.Storage
 			wantProjectNames = append(wantProjectNames, name.FormatProject(pID))
 		}
 		filter := "filters_are_yet_to_be_implemented"
-		gotProjects, _, err := s.ListProjects(filter, 100, "")
+		gotProjects, pageToken, err := s.ListProjects(filter, 100, "")
 		if err != nil {
 			t.Fatalf("ListProjects got %v want success", err)
 		}
+		if pageToken != "" {
+			t.Errorf("ListProjects should return empty page token, got %s", pageToken)
```

Comment:
Generally we do got before want for test errors. E.g.,
Got %s, want empty page token.

Reply:
Thanks @vtsao fixed.